webportal/package.json

```diff
@@ -2,13 +2,25 @@
   "name": "pai-webportal",
   "version": "0.1.0",
   "description": "Webportal for Microsoft Platform for AI",
-  "main": "index.js",
   "keywords": [
     "webportal",
-    "AI"
+    "AI",
+    "platform",
+    "PAI"
```

Comment:
What are these keywords for?

Reply:
For node module search engine optimization.